internal/host/installcmd.go

```diff
@@ -80,6 +80,21 @@ func (i *installCmd) GetStep(ctx context.Context, host *models.Host) (*models.St
 		data["INSTALLATION_TIMEOUT"] = strconv.Itoa(int(i.instructionConfig.InstallationTimeout))
 	}
 
+	if cluster.HTTPProxy != "" || cluster.HTTPSProxy != "" {
+		if cluster.HTTPProxy != "" {
+			cmdArgsTmpl = cmdArgsTmpl + " --http-proxy {{.HTTP_PROXY}}"
+			data["HTTP_PROXY"] = cluster.HTTPProxy
+		}
+		if cluster.HTTPSProxy != "" {
+			cmdArgsTmpl = cmdArgsTmpl + " --https-proxy {{.HTTPS_PROXY}}"
+			data["HTTPS_PROXY"] = cluster.HTTPSProxy
```

Comment:
Why not just concatenate `cluster.HTTPProxy`? Not much value in updating the template and immediately providing a value to be injected, IMO.

Reply:
for consistency with the code above this where the same method is practiced.
if it is desired to be changed, all other places need to be changed as well.